_posts/2024-12-16-vertx-subsystem-in-preview.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "Introducing Vertx Subsystem in WildFly Preview"
+date:   2024-12-16
+tags:   wildfly, vertx, microprofile
+author: gaol
+description: Learn about Vertx configuration in WildFly Preview 35
+---
+
+= Introducing Vertx Subsystem in WildFly Preview
+
+I'm excited to announce the integration of the link:https://github.com/wildfly-extras/wildfly-vertx-feature-pack[WildFly Vertx Feature Pack, window=_blank] into WildFly Preview from WildFly 35 Beta release. This feature pack introduces Vertx configuration capabilities through the WildFly management model, making it easier to manage and integrate Vertx with existing WildFly subsystems.
+
+NOTE: In the 35 release, the vertx subsystem is only available in WildFly Preview, and not in standard WildFly. It is provided at the link:https://docs.wildfly.org/34/Admin_Guide.html#Feature_stability_levels[`preview` stability level, window=_blank], which is enabled out-of-the-box in WildFly Preview.
+
+== Eclipse Vert.x Overview
+
+link:https://github.com/eclipse-vertx/vert.x/[Eclipse Vert.x, window=_blank] is an open-source toolkit designed for building event-driven, asynchronous applications.
+
+Currently, Vertx instances have been used by `opentelemetry` and `reactive-messaging` subsystems within WildFly to provide features powered by vertx components underneath, but there was no central mechanism to configure them.
```

Comment:
The full name of the subsystem is actually 'microprofile-reactive-messaging-smallrye'